test/unit/org/apache/cassandra/cql3/ast/CQLFormatterPrettyPrintTest.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.cql3.ast;
+
+import org.junit.Test;
+
+import org.apache.cassandra.db.marshal.Int32Type;
+import org.apache.cassandra.schema.TableMetadata;
+import org.assertj.core.api.Assertions;
+
+public class CQLFormatterPrettyPrintTest
+{
+    private static TableMetadata TBL1 = TableMetadata.builder("ks", "tbl")
+                                                     .addPartitionKeyColumn("pk", Int32Type.instance)
+                                                     .addClusteringColumn("ck", Int32Type.instance)
+                                                     .addRegularColumn("v0", Int32Type.instance)
+                                                     .addRegularColumn("v1", Int32Type.instance)
+                                                     .build();
+    private static final Symbol pk = new Symbol("pk", Int32Type.instance);
+    private static final Symbol ck = new Symbol("ck", Int32Type.instance);
+    private static final Symbol v0 = new Symbol("v0", Int32Type.instance);
+    private static final Symbol v1 = new Symbol("v1", Int32Type.instance);
+
+    @Test
+    public void selectTest()
+    {
+        Select select = select();
+        Assertions.assertThat(select.toCQL(format())).isEqualTo("SELECT *\n" +
+                                                                "FROM ks.tbl\n" +
+                                                                "WHERE foo = ?\n" +
+                                                                "LIMIT ?");
+    }
+
+    @Test
+    public void insertTest()
+    {
+        Mutation.InsertBuilder builder = Mutation.insert(TBL1);
+        builder.value("pk", 0);
+        builder.value("v1", 0);
+        builder.ttl(new Literal(42, Int32Type.instance));
```

Comment:
by default `Bind` is used when you use raw values.  To make sure this test uses a `Literal` it needs to be explicit.